fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -761,6 +761,7 @@ public void testBinaryEtags() throws IOException, InterruptedException {
         // Next, update the description
         final HttpPatch httpPatch = patchObjMethod(id + "/binary/fcr:metadata");
         httpPatch.addHeader("Content-Type", "application/sparql-update");
+        httpPatch.addHeader("If-Match", descEtag1.replace("W/", ""));
```

Comment:
This should only happen for "W/" at the start of the tag, right? I know it's picky, but can we be precise about that?